src/components/AddressSearch.js

```diff
@@ -86,10 +87,15 @@ const AddressSearch = (props) => {
         <GooglePlacesAutocomplete
             ref={googlePlacesRef}
             fetchDetails
+            onBlur={() => {
+                if(!isSelected){
```

Comment:
This doesn't conform to our style guide - see [here](https://github.com/Expensify/App/blob/main/STYLE.md#whitespace) for more details around whitespace usage.